packages/common/src/config.ts

```diff
@@ -3,11 +3,13 @@ import path from "path";
 import { ClaimerConfig } from "./types";
 
 type CandidateConfig = {
+  id: number;
```

Comment:
this is basically a 1kv slot id, right ? should it be called slotId or similiar ?

Reply:
slotId is probably better, yeah